src/main/java/javaserver/Request.java

```diff
@@ -0,0 +1,40 @@
+package javaserver;
+
+import java.util.Arrays;
+
+public class Request {
+	String method;
+	String URI;
+
+	Request(String method, String URI) {
+		this.method = method;
+		this.URI = URI;
+	}
+
+	public String getMethod() {
+		return method;
+	}
+
+	public String getURI() {
+		return URI;
+	}
+
+	public boolean isOK() {
+		String requestType = getMethod();
+		if (routeOptions() != null && (Arrays.asList(routeOptions()).contains(requestType))) {
+			return true;
+		} else if (this.hasVariableParams()) {
+			return true;
+		} else {
+			return false;
+		}
+	}
+
+	public String[] routeOptions() {
+		return Routes.getOptions(URI);
+	}
+
+	public boolean hasVariableParams() {
```

Comment:
This should really check for the existence for a `?` in the query string if you want to determine if a request contains query parameters. Right now you're tying yourself to only check for parameters at the `/parameter` endpoint.